http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -314,8 +318,15 @@ class GceInterpreter[F[_]: Parallel: ContextShift: Logger](
     if (params.isAsyncRuntimeFields)
       googleComputeService
         .deleteInstance(params.googleProject, config.gceConfig.zoneName, InstanceName(params.runtimeName.asString))
-        .map(x => Some(x))
-    else Async[F].pure(None)
+        .map(x => Option(x))
+        .handleErrorWith {
+          case _: com.google.api.gax.rpc.NotFoundException =>
```

Comment:
Good catch.. probably would be good to have wb-libs `GoogleComputeInterpreter.deleteInstance` recover 404s. But can leave this here for now

Reply:
I thought about adding it in `deleteInstance`, but if I do that, we'll need to return `F[Option[Operation]]` because we won't get a `Operation` when 404...I'm not entirely sure if `F[Option[Operation]]` desirable, so I just added the handler here for now.

Reply:
I see - yeah that recover behavior in wb-libs is typically when it returns `F[Unit]`